app/services/HCAService.d.ts

```diff
@@ -0,0 +1,3 @@
+import { HCAService as service } from './HCAService';
+
+export const HCAService = service;
```

Comment:
This is pretty lazy, but for sake of time I didn't do any actual declarations here. just got an implicit `any` by defining a new constant.